Lib/fontbakery/profiles/googlefonts.py

```diff
@@ -4733,21 +4733,63 @@ def com_google_fonts_check_vertical_metrics(ttFont):
                 f'is {int(hhea_sum*font_upm)} when it should be at least {int(font_upm*1.2)}')
 
     # Check the sum of the hhea metrics is below 2.0
-    elif hhea_sum > 2.0:
+    if hhea_sum > 2.0:
         failed = True
         yield FAIL,\
             Message('bad-hhea-range',
                 'The sum of hhea.ascender+abs(hhea.descender)+hhea.lineGap '
                 f'is {int(hhea_sum*font_upm)} when it should be at most {int(font_upm*2.0)}')
 
     # Check the sum of the hhea metrics is between 1.1-1.5x of the font's upm
-    elif hhea_sum > 1.5:
+    if hhea_sum > 1.5:
```

Comment:
This has the potential to raise both a fail and a warn.

The previous elif approach will fail first if the metrics are mega loose or just rais a warn if they're a little loose which imo seems better.

Reply:
LGTM for everything else.

Reply:
You're right. I had removed all the `elif` statements, but that makes sense only for the newly added checks